ocpBowling/swift/Bowling/bowling_rules.txt

```diff
@@ -0,0 +1,32 @@
+The game consists of 10 frames as shown above.  In each frame the player has
```

Comment:
_nitpick_ if you rename this to `README.md` then it will show in Github nicely formatted when you navigate to the folder